tests/playwright/sanity/modules/ai/assets/js/editor/components/prompt-history/history-button.test.ts

```diff
@@ -0,0 +1,67 @@
+import { test, Page } from '@playwright/test';
+
+import { closeAIDialog, findPromptHistoryButton } from './helper';
+import { userInformationMock } from '../../../../../user-information.mock';
+import WpAdminPage from '../../../../../../../../pages/wp-admin-page';
+
+test.describe( 'AI @ai', () => {
+	const mockRoute = async ( page: Page ) => {
+		await page.route( '/wp-admin/admin-ajax.php', async ( route ) => {
+			const requestPostData = route.request().postData();
+
+			if ( requestPostData.includes( 'ai_get_user_information' ) ) {
+				await route.fulfill( {
+					json: userInformationMock,
+				} );
+			}
+		} );
+	};
+
+	test( 'Prompt History Button', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		const editor = await wpAdmin.openNewPage();
+
+		await mockRoute( page );
+
+		await test.step( 'Textarea control', async () => {
+			await editor.addWidget( 'heading' );
+
+			await findPromptHistoryButton( page );
+
+			await closeAIDialog( page );
+		} );
+
+		await test.step( 'Wysiwyg control', async () => {
+			await editor.addWidget( 'text-editor' );
+
+			await findPromptHistoryButton( page );
+
+			await closeAIDialog( page );
+		} );
+
+		await test.step( 'HTML control', async () => {
+			await editor.addWidget( 'html' );
+
+			await findPromptHistoryButton( page );
+
+			await closeAIDialog( page );
+		} );
+
+		await test.step( 'Image control', async () => {
+			await editor.addWidget( 'image' );
+
+			await findPromptHistoryButton( page );
+
+			await closeAIDialog( page );
+		} );
+
+		await test.step( 'Image box', async () => {
+			await editor.addWidget( 'image-box' );
+
+			await findPromptHistoryButton( page );
+
+			await closeAIDialog( page );
+		} );
```

Comment:
Why do we need these tests in the E2E scope? (instead of a unit test in the Dialog component).

Reply:
It's an internal team agreement that was here before I joined. I've specifically asked how should we cover code with tests and got a confirmation that we're using Playwright.